src/main/java/org/manifold/compiler/front/Main.java

```diff
@@ -252,7 +259,7 @@ public ExpressionVertex visitFunctionInvocationExpression(
     // then get the input vertex
     ExpressionVertex vInput = context.expression(1).accept(this);
     ExpressionEdge eInput = new ExpressionEdge(vInput, null);
-    
```

Comment:
Gah, my attempt to hide another program in whitespace has been foiled!

(tbh - not sure. Might be windows/unix line ending shenanigans? I have intellij set up to remove spaces on write)